File: Otras_Evidencias/arreglos.cpp
// Santiago de Jesús Bolio Cano
//540809

#include <iostream>
#include <array>
using namespace std;
int main()
{
    
    array <int, 10> arreglo; //int arreglo[10];
    
    cout<<"indice" <<"\t" <<"Valor"<< endl;
    //estructura cómun para manipular arreglos es for
    for (int indice = 0; indice < 10; indice++){
        arreglo[indice] = 0;
        cout << indice << "\t" << arreglo[indice] << endl;
    }
    return 0;
}

File: Java/Arreglos.java
// Santiago de Jesús Bolio Cano
//540809


class Arreglos {
  public static void main(String args[]) {
   
   //declaración orientada a objetos
   //clase[]objeto = new Clase[];
   int[] arreglo = new int[10];
   
    System.out.println("indice \tValor");
   
   for(int indice = 0; indice< arreglo.length; indice++){
       System.out.println(indice+"\t"+ arreglo[indice]);
   }
  }
}